spokestack/vad/webrtc.py

```diff
@@ -0,0 +1,74 @@
+"""
+This module contains the webrtc component for
+voice activity detection (vad)
+"""
+
+import webrtcvad  # type: ignore
+
+from spokestack.context import SpeechContext
+
+
+MODE = {"quality": 0, "low_bitrate": 1, "aggresive": 2, "very_aggresive": 3}
+
+
+class VoiceActivityDetector:
+    """ This class detects the presence of voice in a frame of audio.
+
+    :param sample_rate: sample rate of the audio
+    :type sample_rate: int
+    :param frame_width: width of the audio frame
+    :type frame_width: int
+    :param vad_rise_delay: rise delay
+    :type vad_rise_delay: int
+    :param vad_fall_delay: fall delay
```

Comment:
this is not a descriptive comment - it's just rewriting the parameter name with a space